sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventData.java

```diff
@@ -352,9 +352,37 @@ public int hashCode() {
      */
     private static class SystemProperties extends HashMap<String, Object> {
         private static final long serialVersionUID = -2827050124966993723L;
+        private final String offset;
```

Comment:
I thought we were going to make `offset` a numeric;  the service should never send back an offset of "-1" or "@latest" - it should always be a concrete number of bytes.

Reply:
Ah yes. I will create an issue and address this in another PR. thanks for the catch!